euphoria-flink/src/main/java/cz/seznam/euphoria/flink/streaming/InputTranslator.java

```diff
@@ -33,7 +32,6 @@
 
     return context.getExecutionEnvironment()
             .addSource(new DataSourceWrapper<>(ds))
-            .setParallelism(operator.getParallelism())
-            .assignTimestampsAndWatermarks(new IngestionTimeExtractor<>());
+            .setParallelism(operator.getParallelism());
```

Comment:
I am afraid it doesn't. We're still running using `ingestion` time characteristics, not `processing`. So the periodic watermarks will be send.
Also I think testing of `ingestion` or `processing` may be quite tricky because of flipping tests.